torch/csrc/utils/python_dispatch.cpp

```diff
@@ -422,6 +422,17 @@ void initDispatchBindings(PyObject* module) {
             name, overload_name);
       });
 
+  m.def(
+      "_dispatch_maybe_import_abstract_impl_pystub", [](const char* qualname) {
+        // The abstract_impl API doesn't work with overload names yet.
```

Comment:
what the stance on overload names? Banned from pure-python custom ops, but need-to-support for existing C++ custom ops?